opm/parser/eclipse/EclipseState/Aquifer/AquiferConfig.hpp

```diff
@@ -53,6 +53,7 @@ class AquiferConfig {
     bool hasNumericalAquifer() const;
     bool hasAnalyticalAquifer() const;
     const NumericalAquifers& numericalAquifers() const;
+    NumericalAquifers& mutableNumericalAquifers() const;
```

Comment:
Hehe - thank you for naming it `mutable`; that made the intent screaming clear. Where will the callsite actually modifying this object be? 

I must admit my initial reaction to exporting a mutable object in this way is quite skeptical, but we'll see. I realize that due to the dance up and down between common &leftrightarrow; grid &leftrightarrow; simulators it might be difficult to avoid?